Resources/Private/Scss/_breadcrumbs.scss

```diff
@@ -46,30 +47,8 @@
 }
 
 .navigation_menuButton {
-    @include media-l {
-        @include span-l-2;
-    }
-
-    @include media-m {
-        @include span-m-2;
-    }
-
-    @include media-s {
-        @include span-s-2;
-    }
-    display: flex;
-    background: $brand;
-    cursor: pointer;
-    color: $white;
-    justify-content: center;
-    align-items: center;
-    padding: 0;
-    margin: $magic-l/2 0;
-
-    svg {
-        margin-right: $magic-l/4;
-        fill: $white;
-        height: 1em;
-        width: 1em;
-    }
+    @include cols(2);
+    @include button;
+    @include button-color--brand;
+    @include button--in-breadcrumb;
```

Comment:
Do you plan on using button--in-breadcrumb several times? Otherwise I would suggest do style it directly in _breadcrumbs.scss and not via mixin.

Reply:
There are gonna be several "button--in-SOMETHING" (button--in-FORM/SEARCH/INFOBOX/etc.) scenarios and… well you never know if there will be another button block breadcrumbs ;)